examples/speaker_tasks/diarization/conf/inference/diar_infer_meeting.yaml

```diff
@@ -0,0 +1,90 @@
+# This YAML file is created for all types of offline speaker diarization inference tasks in `<NeMo git root>/example/speaker_tasks/diarization` folder.
```

Comment:
How about suggest / warn the scenario of this config file inside it?

Reply:
added a line.